.github/workflows/build-engine-container.yml

```
@@ -1,11 +1,12 @@
 name: build-docker
 on:
-  push:
-    branches:
-      - master
-  release:
-    types:
-      - created
+  # FIXME: 現状masterブランチへのpushでは起動されないので、されるようにする
```

ビルド&プッシュではなく、ビルドのテストが目標だということわかりやすくすると良いかも！
```suggestion
  # FIXME: masterブランチへのpush時にビルド工程をテストする
```